poradnia/stats/tests.py

```diff
@@ -106,6 +115,34 @@ def test_permission_superuser(self):
             self.assertEqual(resp.status_code, 200)
 
 
+class StatsUserRegisteredPermissionTestCase(TestCase):
```

Comment:
Czasem stosuję jeszcze opcję z assertEqual na liście (tj. spodziewanym wynikiem byłoby tutaj `[403, 403, 403]`) - w zasadzie nie wiem czemu tutaj zrobiłem to inaczej.
Rozwiązanie w feder faktycznie wydaje się ciekawsze, planowałem jakoś uporządkować sprawę tych testów, natomiast myślę, że czytelniej będzie to zrobić już w oddzielnym PR.